Makefile

```
@@ -589,7 +589,7 @@ all-builds: cli main-build clean-image $(MERGED_API_SWAGGER_SPEC) $(MERGED_API_S
 main-image: all-builds
 	make docker-build-main-image
 
-kind_available := $(shell command -v kind && kind get clusters | grep -v "No kind clusters found.")
+kind_available := $(shell command -v kind && kind get clusters 2&> /dev/null | grep -v "No kind clusters found.")
```

I don't think this does what you want.
You're probably looking for:
```suggestion
kind_available := $(shell command -v kind && kind get clusters 2> /dev/null)
```
(and the grep is now redundant)

---

Unfortunately this did not work because kind returns status code `0`  when no kind clusters were found.
I've removed the automatic loading of images from the Makefile for now, when I have a better solution I'll re-add this.
Additionally I was not happy because other images were not automatically loaded into kind.